loggingpy/log.py

```diff
@@ -98,6 +110,7 @@ def parse_log_entry(log_entry: LogEntry):
         dto = {
             "timestamp": log_entry.timestamp,
             "level": log_entry.log_level.name,
+            "message": log_entry.message,
```

Comment:
Yes, I can.

Reply:
I was taking it out in .NET because I also create instrumentation data in that doesn't provide any messages, so that would be just noise in potentially frequent data. OTOH, it's no big deal, there's pros too to doing it always. Your call :)